Android build fails with "fatal error: 'androidfonts.c' file not found" when targeting aarch64-linux-android

I'm trying to integrate mupdf-rs into my project using flutter_rust_bridge. Everything builds fine for Linux, but when I attempt to compile for Android, I get the following error:
```
SEVERE: ================================================================================
SEVERE: Cargokit BuildTool failed with error:
SEVERE: --------------------------------------------------------------------------------
SEVERE: External Command: rustup "run" "stable" "cargo" "build" "--manifest-path" "/home/luoyf/Projects/frb/rust/Cargo.toml" "-p" "rust_lib_frb" "--target" "aarch64-linux-android" "--target-dir" "/home/luoyf/Projects/frb/build/rust_lib_frb/build"
SEVERE: Returned Exit Code: 101
SEVERE: --------------------------------------------------------------------------------
SEVERE: STDOUT:
SEVERE: --------------------------------------------------------------------------------
SEVERE: STDERR:
SEVERE: Compiling mupdf-sys v0.7.0
SEVERE:    Compiling rust_lib_frb v0.1.0 (/home/luoyf/Projects/frb/rust)
SEVERE: warning: mupdf-sys@0.7.0: wrapper/context.c:4:10: fatal error: 'androidfonts.c' file not found
SEVERE: warning: mupdf-sys@0.7.0:     4 | #include "androidfonts.c"
SEVERE: warning: mupdf-sys@0.7.0:       |          ^~~~~~~~~~~~~~~~
SEVERE: warning: mupdf-sys@0.7.0: 1 error generated.
SEVERE: error: failed to run custom build command for `mupdf-sys v0.7.0`
SEVERE: 
SEVERE: Caused by:
SEVERE:   process didn't exit successfully: `/home/luoyf/Projects/frb/build/rust_lib_frb/build/debug/build/mupdf-sys-5d85ef0f9017c085/build-script-build` (exit status: 1)
SEVERE:   --- stdout
SEVERE:   TARGET = Some("aarch64-linux-android")
SEVERE:   HOST = Some("x86_64-unknown-linux-gnu")
SEVERE:   cargo:rerun-if-env-changed=CC_aarch64-linux-android
SEVERE:   CC_aarch64-linux-android = Some("/home/luoyf/Android/Sdk/ndk/28.2.13676358/toolchains/llvm/prebuilt/linux-x86_64/bin/clang")
SEVERE:   cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
SEVERE:   CRATE_CC_NO_DEFAULTS = None
SEVERE:   cargo:rerun-if-env-changed=CFLAGS_aarch64-linux-android
SEVERE:   CFLAGS_aarch64-linux-android = Some("--target=aarch64-linux-android24")
SEVERE:   OPT_LEVEL = Some("0")
SEVERE:   cargo:rerun-if-env-changed=CC_aarch64-linux-android
SEVERE:   CC_aarch64-linux-android = Some("/home/luoyf/Android/Sdk/ndk/28.2.13676358/toolchains/llvm/prebuilt/linux-x86_64/bin/clang")
SEVERE:   cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
SEVERE:   CRATE_CC_NO_DEFAULTS = None
SEVERE:   DEBUG = Some("true")
SEVERE:   cargo:rerun-if-env-changed=CFLAGS_aarch64-linux-android
SEVERE:   CFLAGS_aarch64-linux-android = Some("--target=aarch64-linux-android24")
SEVERE:   cargo:rerun-if-env-changed=CXX_aarch64-linux-android
SEVERE:   CXX_aarch64-linux-android = Some("/home/luoyf/Android/Sdk/ndk/28.2.13676358/toolchains/llvm/prebuilt/linux-x86_64/bin/clang++")
SEVERE:   cargo:rerun-if-env-changed=CXXFLAGS_aarch64-linux-android
SEVERE:   CXXFLAGS_aarch64-linux-android = Some("--target=aarch64-linux-android24")
```

I've already disabled all default features:
toml

`mupdf = { version = "0.7.0", default-features = false }`

The issue persists. Any suggestions? Thanks.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -57,7 +57,7 @@ sys-lib-brotli = ["mupdf-sys/sys-lib-brotli"]
 # plain PDFs render without text unless replacement fonts are provided at
 # runtime, e.g. via a `FontLoader`.
 base14-fonts = ["mupdf-sys/base14-fonts"]
-system-fonts = ["font-kit"]
+system-fonts = ["dep:font-kit"]
 
 # Runtime bundled font providers. These load fonts through MuPDF's system font
 # hooks in the safe crate instead of compiling them into mupdf-sys.
@@ -103,7 +103,7 @@ mupdf-fonts-droid = { version = "0.8.0", path = "mupdf-fonts-droid", optional = 
 mupdf-fonts-noto = { version = "0.8.0", path = "mupdf-fonts-noto", optional = true }
 mupdf-fonts-sil = { version = "0.8.0", path = "mupdf-fonts-sil", optional = true }
 
-[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
+[target.'cfg(all(not(target_arch = "wasm32"), not(target_os = "android")))'.dependencies]
 font-kit = { version = "0.14.1", optional = true }
 
 [workspace]

--- a/src/font_loader.rs
+++ b/src/font_loader.rs
@@ -80,7 +80,11 @@ pub fn set_font_loader(loader: impl FontLoader) {
 const BUILT_IN_LOADERS: &[&dyn FontLoader] = &[
     #[cfg(feature = "bundled-fonts-runtime")]
     &crate::bundled_font::BundledFontLoader,
-    #[cfg(all(feature = "system-fonts", not(target_arch = "wasm32")))]
+    #[cfg(all(
+        feature = "system-fonts",
+        not(target_arch = "wasm32"),
+        not(target_os = "android")
+    ))]
     &crate::system_font::SystemFontLoader,
 ];
 

--- a/src/system_font.rs
+++ b/src/system_font.rs
@@ -2,8 +2,9 @@
 //!
 //! Each shim translates MuPDF's raw callback arguments into safe types and
 //! dispatches to the [`FontLoader`](crate::FontLoader) chain (the registered
-//! user loader first, then the built-in loaders). `SystemFontLoader` is the
-//! built-in loader backed by `font-kit` for fonts installed on the system.
+//! user loader first, then the built-in loaders). On non-Android, non-wasm
+//! targets, `SystemFontLoader` is the built-in loader backed by `font-kit` for
+//! fonts installed on the system.
 
 use std::ffi::CStr;
 use std::os::raw::{c_char, c_int};
@@ -102,11 +103,19 @@ pub(crate) unsafe extern "C" fn load_system_fallback_font(
 }
 
 /// Looks up fonts installed on the system via `font-kit`.
-// `font-kit` is only a dependency on non-wasm targets.
-#[cfg(all(feature = "system-fonts", not(target_arch = "wasm32")))]
+// `font-kit` is only a dependency on non-Android, non-wasm targets.
+#[cfg(all(
+    feature = "system-fonts",
+    not(target_arch = "wasm32"),
+    not(target_os = "android")
+))]
 pub(crate) struct SystemFontLoader;
 
-#[cfg(all(feature = "system-fonts", not(target_arch = "wasm32")))]
+#[cfg(all(
+    feature = "system-fonts",
+    not(target_arch = "wasm32"),
+    not(target_os = "android")
+))]
 impl font_loader::FontLoader for SystemFontLoader {
     fn load_font(&self, name: &str, hints: FontHints) -> Option<Font> {
         use font_kit::family_name::FamilyName;

